tracks/rust/exercises/reverse-string/mentoring.md

```diff
@@ -0,0 +1,111 @@
+### Concepts
+
+- &str
```

Comment:
You might want to consider linking to the official docs, so mentors have a quick reference. This goes for all these and all the method calls in this document.

Reply:
Great idea! I only linked to the methods that we want mentors/students to use :)